js/pages/settings.js

```diff
@@ -760,9 +795,9 @@ export const renderChangeContactInformationGroup = userData => {
                           </label>
                           <br>
                           <div class="btn-group col-md-4" id="editHomePhone">
-                            <input type="tel" class="form-control num-val-phone" value="${userData[cId.homePhone].substr(0, 3)}" id="homePhoneNumber1" data-val-pattern="[1-9]{1}[0-9]{2}" title="Only numbers are allowed." size="3" maxlength="3" Placeholder="999"> <span class="hyphen">-</span>
-                            <input type="tel" class="form-control num-val-phone" value="${userData[cId.homePhone].substr(3, 3)}" id="homePhoneNumber2" data-val-pattern="[0-9]{3}" title="Only numbers are allowed." size="3" maxlength="3" Placeholder="999"> <span class="hyphen">-</span>
-                            <input type="tel" class="form-control num-val-phone" value="${userData[cId.homePhone].substr(6, 4)}" id="homePhoneNumber3" data-val-pattern="[0-9]{4}" title="Only numbers are allowed." size="4" maxlength="4" Placeholder="9999">
+                            <input type="tel" class="form-control num-val-phone" value="${userData[cId.homePhone] && userData[cId.homePhone].length === 10 ? userData[cId.homePhone].substr(0, 3) : ''}" id="homePhoneNumber1" data-val-pattern="[1-9]{1}[0-9]{2}" title="Only numbers are allowed." size="3" maxlength="3" Placeholder="999"> <span class="hyphen">-</span>
+                            <input type="tel" class="form-control num-val-phone" value="${userData[cId.homePhone] && userData[cId.homePhone].length === 10 ? userData[cId.homePhone].substr(3, 3) : ''}" id="homePhoneNumber2" data-val-pattern="[0-9]{3}" title="Only numbers are allowed." size="3" maxlength="3" Placeholder="999"> <span class="hyphen">-</span>
```

Comment:
if these rules are exactly the same (home, cell, other) possibly helpful to just create a function that validates phone numbers?

Reply:
I included phone validation in `settings.js` -> `validateContactInformation()` but was cautions about further combining since visibility can change depending on whether the data exists or is null. Happy to combine these if you'd like.